sdk/openai/Azure.AI.OpenAI/tests/AudioSpeechTests.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.AI.OpenAI.Tests;
+
+public class AudioSpeechTests : OpenAITestBase
+{
+    public AudioSpeechTests(bool isAsync)
+        : base(Scenario.AudioSpeech, isAsync) // , RecordedTestMode.Live)
+    {
+    }
+
+    [RecordedTest]
+    [TestCase(Service.Azure)]
+    [TestCase(Service.NonAzure)]
+    public async Task GetAudioSpeech(Service serviceTarget)
+    {
+        OpenAIClient client = GetTestClient(serviceTarget);
+        string deploymentOrModelName = GetDeploymentOrModelName(serviceTarget);
+
+        var requestOptions = new AudioSpeechOptions(
+            deploymentOrModelName, "Hello World", AudioSpeechVoice.Alloy
+        );
+
+        Response<BinaryData> response = await client.GetAudioSpeechAsync(requestOptions);
+        Assert.That(response, Is.Not.Null);
```

Comment:
Suggest to change to:

`Assert.That(response?.Value, Is.Not.Null);`

Reply:
Resolved by https://github.com/ninghu/azure-sdk-for-net/pull/7/commits/a363fc9836624a1dfb1338631461a4c6471c5b02